### PR TITLE
content-management-policy-application-to-files issues-18

### PR DESCRIPTION
**Pull request by jaind1046**
Policy-application-to-file

### CODE REVIEW

File: specs/step_definitions/content-management-policy-application-to-files.steps.js [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+    I.onIcapProxyPage()
+    icapProxyPage.downloadFile(fileType)
+    I.wait(15)
+    icapProxyPage.assertFileDownload(file)
+})
+
+Then(/^The (.*) processing outcome is (.*)$/, (file, fileOutcome) => {
+    I.loginNoPwd();
+    homePage.clickFileDrop()
+    I.uploadFileWithNoSanitiseData(`output/downloads/${file.trim()}`)
```
Comment (NadHodeGW) [inline specs/step_definitions/content-management-policy-application-to-files.steps.js:27]: where is assertion here that shows the file outcome?

File: src/utils/steps_file.js [hunk 2]
```diff
@@ -63,6 +63,12 @@ module.exports = function () {
             this.waitForElement(filedropPage.sections.analysisReportView,30)
         },
 
+        uploadFileWithNoSanitiseData: function (file) {
+            this.attachFile(filedropPage.buttons.fileInput, file)
+            this.waitForElement(filedropPage.buttons.refresh,60)
+            this.dontSeeElement(filedropPage.sections.analysisReportView)
```
Comment (NadHodeGW) [inline src/utils/steps_file.js:69]: why are you putting assertions in the step file? please remove any i.see or dont see from here. also, in function  uploadFileWithNoSanitiseData what are you trying to check by saying: this.dontSeeElement(filedropPage.sections.analysisReportView)?

**Review by NadHodeGW — CHANGES_REQUESTED** [review]
Please see comment in files

File: src/utils/steps_file.js (hunk 2 above)
Comment (jaind1046) [reply src/utils/steps_file.js:69]: Removed this dontSeeElement instead of it just clicked the download.

**Comment by jaind1046** [comment]
https://drive.google.com/file/d/1Du_ZskBIHQlBgXewRD-TvswF6eTAQlsg/view?usp=sharing

**Review by NadHodeGW — APPROVED** [review]

**Pull request merged**